relayer/middleware/wallet/wallet-management.ts

```diff
@@ -62,10 +62,12 @@ const networks = {
 };
 
 export type PrivateKeys = Partial<{ [k in ChainId]: string[] }>;
+export type TokenAddresses = Partial<{ [k in ChainId]: string[] }>;
```

Comment:
this is a bit confusing to me because we are not really passing token addresses, but token names. Wouldn't it be better to use `TokenNames` or something of the like? (I like TokenNamesByChain)

Reply:
I went with `TokensByChain`